Makefile

```diff
@@ -9,7 +9,6 @@ DSWIFI_MAJOR	:= 0
 DSWIFI_MINOR	:= 4
 DSWIFI_REVISION	:= 2
 VERSION		:= $(DSWIFI_MAJOR).$(DSWIFI_MINOR).$(DSWIFI_REVISION)
-VERSION_HEADER	:= include/dswifi_version.h
```

Comment:
I think we should just get rid of all the version definitions in the Makefile. This is a leftover of the time when I ported the build system to BlocksDS, they aren't really needed for anything. We can keep them in the header only.

Reply:
Yea, I wasn't sure if the version numbers still served a purpose in the Makefiles. Latest commit removes them